server/tserver/src/main/java/org/apache/accumulo/tserver/log/DfsLogger.java

```diff
@@ -417,6 +417,13 @@ public synchronized void open(String address) throws IOException {
       else
         logFile = fs.create(logfilePath, true, 0, replication, blockSize);
 
+      // Tell the DataNode that the write ahead log does not need to be cached in the OS page cache
+      try {
+        logFile.setDropBehind(Boolean.TRUE);
```

Comment:
It's weird that the API parameter is a nullable `Boolean` and not the primitive `boolean`. I see that it is marked `@InterfaceStability.Evolving`, but it has been present since at least 3.0.0, so I think it's probably fine.

Reply:
Javadoc says that if parameter is null, then it defaults to configuration, which should be the value of `dfs.datanode.drop.cache.behind.writes`